services/web/server/src/simcore_service_webserver/projects/_controller/_rest_utils.py

```diff
@@ -5,9 +5,9 @@
 from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
 from servicelib.rest_constants import RESPONSE_MODEL_POLICY
 
-from . import _permalink_service
-from ._crud_api_read import _paralell_update
-from .models import ProjectDict
+from .. import _permalink_service
+from .._crud_api_read import _paralell_update
```

Comment:
is this one not a service then?

Reply:
Yes, it's a service. As mentioned, I skipped some tightly coupled modules (like `_crud_api*`) in this PR to avoid complicating the transition. Next, I'll split them to reduce coupling, then convert them into the service layer.